db/migrate/20190926234420_create_questions.rb

```diff
@@ -1,6 +1,7 @@
 class CreateQuestions < ActiveRecord::Migration[6.0]
   def change
     create_table :questions do |t|
+      t.integer :chart_id
```

Comment:
We can just remove this line and not have the `RemoveChartIdFromQuestions` migration.

If your local database has already consumed this migration. You'll have to run `rails db:reset`.

Reply:
I know I said it was poor practice above but since we're still in the process of setting everything up it's fine.

Reply:
Just once we have the environment in a good state, we'll then need to not go back and change history.